### PR TITLE
TIG-3503: Make it possible to run a genny workload from a private repo

### PR DESCRIPTION
**Pull request by mridul-augustine**

### CODE REVIEW

**Review by rtimmons — APPROVED** [review]

File: src/lamplib/src/genny/tasks/auto_tasks.py [hunk 1]
```diff
@@ -70,23 +70,34 @@ class WorkloadLister:
...
         self.genny_repo_root = genny_repo_root
         self._expansions = None
         self.reader = reader
 
     def all_workload_files(self) -> Set[str]:
-        pattern = os.path.join(self.genny_repo_root, "src", "workloads", "**", "*.yml")
+        pattern = os.path.join(self.workspace_root, "src", "*", "src", "workloads", "**", "*.yml")
         return {*glob.glob(pattern)}
 
     def modified_workload_files(self) -> Set[str]:
```
Comment (aggrand) [inline src/lamplib/src/genny/tasks/auto_tasks.py:83]: Good catch on the modified workloads!

**Review by aggrand — APPROVED** [review]
LGTM % a green patch showing the currently-scheduled autorun tasks still get scheduled

**Comment by mridul-augustine** [comment]
evergreen merge

**Comment by mridul-augustine** [comment]
evergreen merge --module dsi:975

**Pull request merged**